internal/pkg/manifest/pipeline.go

```diff
@@ -9,10 +9,12 @@ import (
 	"fmt"
 	"text/template"
 
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/cloudformation"
```

Comment:
We could get rid of this dependency being forced on clients by changing the `StackConfiguration` interface abstraction to deal with strings instead of `cloudformation.Parameters` and `cloudformation.Tags`, and internally to the `cloudformation` package translate the input strings into the `service/cloudformation` package specific values hiding that from consumers.

I don't think that should be taken on as part of this PR, but I think we should consider doing it at some point.

Reply:
I moved all CFN serialization logic into the cloudformation package and since we don't seem to have the intent to make `stackConfiguration` a public interface, it makes less sense to change the return types of `Parameters()` and `Tags()`.